CONTRIBUTING.md

```diff
@@ -0,0 +1,305 @@
+
+If model uses dataset which is unsupported by Accuracy Checker, you also must provide link to it. Please notice this issue in PR description. Don't forget about dataset license too (see [above](#how-to-contribute-model-to-open-model-zoo)).
+
+When the configuration file is ready, you must run Accuracy Checker to obtain metric results. If they match your results, that means  conversion was fully successful and Accuracy Checker fully supports your model, metric and dataset. If no - recheck [conversion](#model-conversion) parameters or validation configuration file.
+
+*After this step you will get accuracy validation configuration file - **<model_name>.yml***
+
+### Example
+
+Let use one of the file from `tools/accuracy_checker/configs`, for example, this is validation configuration file for [AlexNet](tools/accuracy_checker/configs/alexnet.yml):
+
+```
+models:
+  - name: alexnet-cf
+    launchers:
+      - framework: caffe
+        model:   public/alexnet/alexnet.prototxt
+        weights: public/alexnet/alexnet.caffemodel
+        adapter: classification
+    datasets:
+      - name: imagenet_1000_classes
+        preprocessing:
+          - type: resize
+            size: 256
+          - type: crop
+            size: 227
+          - type: normalization
+            mean: 104, 117, 123
+
+  - name: alexnet
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/alexnet/FP32/alexnet.xml
+        weights: public/alexnet/FP32/alexnet.bin
+        adapter: classification
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   public/alexnet/FP16/alexnet.xml
+        weights: public/alexnet/FP16/alexnet.bin
+        adapter: classification
+
+    datasets:
+      - name: imagenet_1000_classes
+        preprocessing:
+          - type: resize
+            size: 256
+          - type: crop
+            size: 227
+
+        metrics:
+          - name: accuracy@top1
+            type: accuracy
+            top_k: 1
+          - name: acciracy@top5
+            type: accuracy
+            top_k: 5
+```
+
+
+## Documentation
+
+Documentation is very important part of model contribution, it helps to better understand possible usage of the model. Documentation must be named after suggested models name.
+Documentation should contain:
+* description of model
+	* main purpose
+	* features
+	* links to paper or/and source
+* model specification
+	* type
+	* framework
+	* GFLOPs (*if available*)
+	* number of parameters (*if available*)
+* validation dataset description/link
+* main accuracy values (also description of metric)
+* detailed description of input and output for original and converted models
+
+Detailed structure and headers naming convention you can learn from any other model, e.g. [alexnet](./models/public/alexnet/alexnet.md).
```

Comment:
`Detailed structure and headers naming convention you can learn from any other model` -> `You can learn (or just Learn) a detailed structure and headers...`